Makefile

```diff
@@ -1,16 +1,22 @@
-.PHONY: all image package dist clean
+.PHONY: all image package dist deploy clean
 
-all: package
+all: image dist
 
 image:
 	docker build --tag amazonlinux:nodejs .
 
-package: image
-	docker run --rm --volume ${PWD}/lambda:/build amazonlinux:nodejs npm install --production
+awsconfigure:
+	docker run -it --rm --volume ${PWD}/../.aws:/root/.aws amazonlinux:nodejs aws configure
```

Comment:
agreed, it will reduce the overall complexity of steps to get started with the the project. I will commit changes later